packages/yoroi-extension/app/components/topbar/NavWalletDetails.js

```diff
@@ -121,7 +123,7 @@ export default class NavWalletDetails extends Component<Props> {
               className={styles.toggleButton}
               onClick={onUpdateHideBalance}
             >
-              {shouldHideBalance ? <IconEyeClosed /> : <IconEyeOpen />}
+              {!showEyeIcon ? null : shouldHideBalance ? <IconEyeClosed /> : <IconEyeOpen />}
```

Comment:
When `showEyeIcon === true`, the whole `<button>` should not be rendered instead of rendering an empty button.

Reply:
Great note